workspace/test/test-core/test-defer-pulse/patch.test.tsv

```diff
@@ -0,0 +1,13 @@
+IN	OUT
+no-pulse	no-pulse
+
+pulse	no-pulse
+no-pulse	pulse
+
+error	no-pulse
+no-pulse	error
+
+pulse	error	// is this okay?
```

Comment:
If we’d talk about pulses as of type with values now (pulse) / past (no-pulse) / error (error), the line above the current (no-pulse) sets it to “past”. So, I’d expect `no-pulse` instead of the `error` here.

Does it conflict with any other reasoning?